src/helpers/Auth.js

```diff
@@ -15,4 +15,28 @@ export async function doLogin(email: string, password: string) {
     return '';
   }
 }
-export const getCookieToken = () => Cookies.get(COOKIE_LOGIN_KEY);
+
+export async function socialLogin(provider: string) {
+  const location = window.location.href.replace(
```

Comment:
This social logins should work on `introduce-smart-faq.fe.staging.kiwi.com/` domain, if you want to try locally pls change this line for `https://introduce-smart-faq.fe.staging.kiwi.com/en/`